src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,51 @@
+.calendar {
+  display: flex;
+  flex-flow: row wrap;
+  gap: 1px 1px;
+
+  width: $columns-quantity * $square-size + 20px;
+  padding: 10px;
+
+  &__day {
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    box-sizing: border-box;
+
+    height: $square-size;
+    width: $square-size;
+
+    cursor: pointer;
+
+    background-color: #eee;
+    border: 1px solid black;
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+
+    transition: background-color 0.5s, transform 0.5s;
```

Comment:
👍 
You can also create variable for transition-duration

Reply:
One variable solely for time for both values, or what exactly?